xray/commands/audit/mvn_test.go

```diff
@@ -19,6 +19,9 @@ func TestMavenTreesMultiModule(t *testing.T) {
 
 	// Check root module
 	multi := getAndAssertNode(t, modulesDependencyTrees, "org.jfrog.test:multi:3.7-SNAPSHOT")
+	if multi == nil {
+	    t.Fatal(err)
```

Comment:
This err seems unrelated to this assertion. It is not returned by it. It is returned from a previous API.